.github/workflows/template-setup-e2e-test/action.yaml

```diff
@@ -39,11 +39,11 @@ runs:
     - name: Setup Minikube Cluster
       uses: medyagh/setup-minikube@v0.0.14
       with:
-        network-plugin: cni
-        cni: flannel
+        # network-plugin: cni
+        # cni: flannel
```

Comment:
We need to set up CNI due to https://github.com/kubernetes/minikube/pull/16143.

Reply:
Alright 22.04 work, let me submit separate PR to update the ubuntu version for our E2Es.

Reply:
Thank you 🎉